debug_missingAnnotationForCallable
debug_nonUniqueAnnotationForCallable
debug_missingAnnotationForCall
expectedCallEdgeNotFound
| code/class_simple.py:50:1:50:15 | Attribute() | code/class_simple.py:8:5:8:26 | Function some_method |
| code/class_simple.py:52:1:52:21 | Attribute() | code/class_simple.py:13:5:13:28 | Function some_staticmethod |
| code/class_simple.py:54:1:54:20 | Attribute() | code/class_simple.py:18:5:18:30 | Function some_classmethod |
| code/class_simple.py:57:1:57:21 | Attribute() | code/class_simple.py:13:5:13:28 | Function some_staticmethod |
| code/class_simple.py:59:1:59:20 | Attribute() | code/class_simple.py:18:5:18:30 | Function some_classmethod |
| code/class_simple.py:83:1:83:15 | Attribute() | code/class_simple.py:65:5:65:26 | Function some_method |
| code/class_simple.py:90:1:90:15 | Attribute() | code/class_simple.py:75:5:75:26 | Function some_method |
unexpectedCallEdgeFound
